### PR TITLE
#94 - Inconsistent behavior for assume_sparse mode

### LINKED ISSUE #94: Inconsistent behavior for assume_sparse mode
Compare [this block in reach](https://github.com/JuliaReach/Reachability.jl/blob/master/src/ReachSets/reach.jl#L70) with [this block in check_property](https://github.com/JuliaReach/Reachability.jl/blob/master/src/Properties/check_property.jl#L66).

The further crashes for the MNA5 model.

Which one should we use?

### PR DESCRIPTION
**Pull request by schillic**
Closes #94.

I did not add a `preprocess` function because it was a one-liner.

### CODE REVIEW

File: src/solve.jl [hunk 1]
```diff
@@ -128,6 +128,13 @@ function solve(system::AbstractSystem,
         transformation_matrix = nothing
     end
 
+    # ==============================
+    # Sparse/dense matrix conversion
+    # ==============================
+    try
+        Δ = options[:assume_sparse] ? sparse(S.A) : full(S.A)
+    end
```
Comment (mforets) [inline src/solve.jl]: isn't a `catch .. Δ = S.A` missing? i'm not sure what happens if the given `AbstractMatrix` doesn't implement `full` or `sparse`? we should check what happens with the `SparseMatrixExp` case.

Comment (schillic) [reply src/solve.jl]: `try error("") end` works.

Comment (schillic) [reply src/solve.jl]: `Δ` is already defined - it would just keep its value.

Comment (schillic) [reply src/solve.jl]: Oh my, what did I do here? This was complete nonsense. Not it is fixed.

**Pull request merged**